test/test_numbers.py

```diff
@@ -0,0 +1,52 @@
+import os
+import unittest
+from data_highlight.highlighter import  HighlightedFile
+
+path = os.path.abspath(__file__)
+dir_path = os.path.dirname(path)
+TEST_FILE = os.path.join(dir_path, "reptest1.rep")
+ERROR_TEST_FILE = os.path.join(dir_path, "reptest2.rep")
+
+
+DATA_FILE = 'data_highlight/file.txt'
+
+class SimpleTest(unittest.TestCase):
+    ############################
+    #### setup and teardown ####
+    ############################
+
+    def setUp(self):
+        pass
+
+    def tearDown(self):
+        pass
+
+    ####################
+    #### file tests ####
+    ####################
+
+    def test_number_of_lines(self):
+
+        dataFile = HighlightedFile('data_highlight/file.txt',2)

```

Comment:
we don't need to use this string filename when we have a constant for it at line 11.

Reply:
Oh okay , I didn't mention contstant.

Reply:
Okay Got it